featuregate/controller/cmd/manager/main.go

```diff
@@ -17,8 +17,8 @@ import (
 	// +kubebuilder:scaffold:imports
```

Comment:
question: not sure what would be the best practice about the location of `main.go`. Just want to learn a little bit on why we want to put it under `cmd/manager` instead of under `controller` directly ? We might want to check with other modules to make things consistent ?

Reply:
Generally binaries are put in `cmd/` in a lot of Go codebases. But yes, we should make it consistent.

Reply:
it made sense to put binaries under `cmd` because previously we had multiple controllers and cli plugins in single go module, now that we are refactoring and creating multiple go modules, not sure if it still makes sense to put the main.go under `cmd/manager`. I'm inclined to putting them under `controller` directory and if you look at kubebuilder it scaffolds the main.go in the project's root directory.